cmd/spark-connect-example-spark-session/main.go

```diff
@@ -21,6 +21,8 @@ import (
 	"flag"
 	"log"
 
+	"github.com/apache/spark-connect-go/v35/spark/sql/utils"
```

Comment:
There should not be spaces between these imports

Reply:
This comes from the gofumpt rules serparating the std library from external packages

<img width="977" alt="image" src="https://github.com/apache/spark-connect-go/assets/3421/6319551c-587a-4888-a7bf-2a1d81543ce8">

Reply:
Apologies for being a bit imprecise. I mean this import and the two below that also not std lib and are from the same library

Reply:
Will fix that.